alembic/versions/5e09fc587903_add_total_room_feedback_views_column.py

```diff
@@ -0,0 +1,28 @@
+"""add total_room_feedback_views_column
```

Comment:
Ensure to run a fresh migration from the develop branch to avoid multiple head errors

Reply:
I have updated the alembic file @Bonifase.